packages/components/src/components/dropdown-list/dropdown-list.tsx

```diff
@@ -107,8 +144,14 @@ const ListItems = React.forwardRef((props, ref) => {
 });
 ListItems.displayName = 'ListItems';
 
-const DropdownList = React.forwardRef((props, ref) => {
-    const { dropdown_ref, list_item_ref, list_wrapper_ref } = ref;
+type TRef = {
+    dropdown_ref: React.RefObject<HTMLDivElement>;
+    list_item_ref: React.RefObject<HTMLDivElement>;
+    list_wrapper_ref: React.RefObject<HTMLDivElement>;
+};
+
+const DropdownList = React.forwardRef<TRef, TDropDownList>((props, ref) => {
+    const { dropdown_ref, list_item_ref, list_wrapper_ref } = ref as unknown as TRef;
```

Comment:
@niloofar-deriv This doesn’t seem to be the right way to use `forwardRef`, We are passing 3 different refs, Maybe we can just use props for this.

Reply:
Yeah you are right but this is not in the scope of this card and by changing this we may have side effects